internal/pkg/term/progress/ecs.go

```diff
@@ -180,6 +200,98 @@ func (c *rollingUpdateComponent) renderAlarms(out io.Writer) (numLines int, err
 	return renderComponents(out, components)
 }
 
+type stoppedTasksInfo struct {
```

Comment:
Can we move the definition of this struct inside of `renderStoppedTasks`, since it's the only place where this is used?